invenio_app_rdm/records_ui/views/records.py

```diff
@@ -100,10 +100,11 @@ class PreviewFile:
     `invenio_previewer.api.PreviewFile`.
     """
 
-    def __init__(self, file_item, record_pid_value, url=None):
+    def __init__(self, file_item, record, record_pid_value, url=None):
         """Create a new PreviewFile."""
         self.file = file_item
         self.data = file_item.data
+        self.record = record
```

Comment:
```suggestion
    def __init__(self, file_item, record_pid_value, record=None, url=None):
        """Create a new PreviewFile."""
        self.file = file_item
        self.data = file_item.data
        self.record = record
```

To avoid breaking any existing usage of the class, we should use kwargs.